substrate/client/authority-discovery/src/worker.rs

```diff
@@ -168,6 +179,16 @@ pub struct Worker<Client, Block: BlockT, DhtEventStream> {
 	phantom: PhantomData<Block>,
 }
 
+#[derive(Debug, Clone)]
+struct RecordInfo {
+	// Time since UNIX_EPOCH in nanoseconds.
+	creation_time: u128,
+	// Peers that we know have this record.
+	peers_with_record: HashSet<PeerId>,
+	// The record itself.
+	record: Record,
```

Comment:
```suggestion
	/// Time since UNIX_EPOCH in nanoseconds.
	creation_time: u128,
	/// Peers that we know have this record.
	peers_with_record: HashSet<PeerId>,
	/// The record itself.
	record: Record,
```